extensions/applicationinsights-react-native/src/ReactNativePlugin.ts

```diff
@@ -45,9 +53,23 @@ export class ReactNativePlugin implements ITelemetryPlugin {
                     this._config[option]
                 );
             }
+
             if (!this._config.disableDeviceCollection) {
                 this._collectDeviceInfo();
             }
+
+            if (extensions) {
+                CoreUtils.arrForEach(extensions, ext => {
+                    const identifier = (ext as ITelemetryPlugin).identifier;
+                    if (identifier === 'ApplicationInsightsAnalytics') {
```

Comment:
Can we define this as an exported constant (like the PropertiesPluginIdentifier) so that we don't get into a situation where the name is changed on the plugin and this just stops working.